### DISCUSSION TITLE
How can insert a varables inside code block output?

### DISCUSSION THREAD

**Discussion by eggcaker** [fonsp/Pluto.jl#1547] [Questions about using Pluto] [answered]
ex: I defined a variable in another block
```
userId = 1000
```
if I write a markdown code block like 
```
md"""
- $userId
"""
```

it's can render to 

```- 1000```
but if I insert a $userId to code block, the userId not render to 1000

```
md"""
’‘’java
public string userId = $userId;
‘’‘
"""
```

**Comment by MichaelHatherly** [comment, marked as answer]
Interpolating into code blocks and code literals inside `md"` strings isn't supported. This was a design choice of the `Markdown` parser itself rather than anything related to `Pluto`. If you're wanting to interpolate into code literals like that then you can call `Markdown.parse` on a `String` where you have already interpolated `userID`:

~~~julia
Markdown.parse(
	"""
	```java
	public string userId = $userId;
	```
	"""
)
~~~

**Reply by eggcaker** [reply]
it's works!
